processing/src/test/java/io/druid/query/groupby/epinephelinae/ByteBufferMinMaxOffsetHeapTest.java

```diff
@@ -204,8 +204,8 @@ public void testRemove()
     ByteBuffer myBuffer = ByteBuffer.allocate(1000000);
     ByteBufferMinMaxOffsetHeap heap = new ByteBufferMinMaxOffsetHeap(myBuffer, limit, Ordering.<Integer>natural(), null);
 
-    for (int i = 0; i < values.size(); i++) {
-      heap.addOffset(values.get(i));
+    for (Integer value : values) {
```

Comment:
It's iteration over `ArrayList<Integer>` so boxed already. Also it's in test so I didn't care about performance